api/src/main/java/jakarta/enterprise/inject/build/compatible/spi/BuildServicesResolver.java

```diff
@@ -61,4 +62,12 @@ private static void discoverFactories() {
 
         BuildServicesResolver.discoveredBuildServices = Collections.unmodifiableSet(factories);
     }
+
+    /**
+     * This method should <em>not</em> be used by applications. It is only exposed for integrators
+     * with complex classloading architectures, where service loader lookup doesn't work out of the box.
```

Comment:
As a public method, also needs the documentation for the parameter. Javadoc lint will likely also complain about this.

Reply:
Good point too.